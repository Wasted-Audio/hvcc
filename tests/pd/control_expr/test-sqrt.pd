#N canvas 827 239 734 565 12;
#X obj 40 22 loadbang;
#X obj 40 49 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 40 230 print;
#X msg 40 137 0.5;
#X obj 40 194 expr sqrt($f1);
#X msg 87 137 3.7;
#X msg 133 137 -5;
#X obj 40 80 t b b b;
#X connect 0 0 1 0;
#X connect 1 0 7 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 4 0;
#X connect 6 0 4 0;
#X connect 7 0 3 0;
#X connect 7 1 5 0;
#X connect 7 2 6 0;
