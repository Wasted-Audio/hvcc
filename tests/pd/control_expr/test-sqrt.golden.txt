[@ 0.000] print: -nan
[@ 0.000] print: 1.92354
[@ 0.000] print: 0.707107
